angular/projects/demo/src/app/init.component.html

```diff
@@ -0,0 +1,9 @@
+<label>
+  <input type="checkbox" [(ngModel)]="isSwitchedOn"> {{ isSwitchedOn ? 'EDIT' : 'VIEW' }}
+</label>
+<div *ngIf="isSwitchedOn; else elseBlock">
+  <app-root></app-root>
```

Comment:
so you're not just reloading the grid over, but destroying entire app and grid each time.